understanding/22/focus-appearance.html

```diff
@@ -61,79 +65,265 @@ <h2>In brief</h2>
     </section>
 
     <section id="intent">
-        
+
         <h2>Intent of Focus Appearance</h2>
 
 
         <p>The purpose of this Success Criterion (SC) is to ensure a keyboard focus indicator is clearly visible and discernible. Focus Appearance is closely related to <a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html">2.4.7 Focus Visible</a> and <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">1.4.11 Non-text Contrast</a>. Focus Visible requires that a visible focus indicator exists while a component has keyboard focus; this SC defines a minimum level of visibility. Where Non-text Contrast requires a component to have adequate contrast against the background in each of its states, this SC requires sufficient contrast for the focus indicator itself.</p>
```

Comment:
Are we sure we want to abbreviate success criterion?

Reply:
There are a few other cases where an understanding doc does this, but most of them don't, so I agree it'd be better to avoid it for consistency. Dropped the abbreviation, replaced it with "Success Criterion" in most cases, replaced with "Focus Appearance" in cases where it was being directly compared with the titles of other related criteria.